internal/boxcli/midcobra/midcobra.go

```diff
@@ -6,6 +6,8 @@ package midcobra
 import (
 	"context"
 	"encoding/hex"
+	"errors"
```

Comment:
If you change this to `github.com/pkg/errors`, does that fix the go sum issues?

Reply:
I'll check

Reply:
@LucilleH I rebased changes onto the latest main, and seems resolved :)